### PR TITLE
ONNX add tests against ORT

### PR DESCRIPTION
**Pull request by geohotstan**
split from #9609 
 - added missing test to cover Squeeze op when axes is None
 - resolved PRelu to match ORT
 - added test coverage for tests in `test/external/external_test_onnx_ops.py` where both we and ORT fails at but we match ORT behavior

### CODE REVIEW

File: extra/onnx.py [hunk 1]
```diff
@@ -290,13 +290,8 @@ def Softmax_13(x:Tensor, axis:int=-1): return x.softmax(axis)
...
   def Gelu(x:Tensor, approximate:str|None=None): return x.gelu() if approximate == "tanh" else 0.5 * x * (1 + (x/math.sqrt(2)).erf())
   def BiasGelu(x: Tensor, bias: Tensor, approximate: str | None = None) -> Tensor: return Gelu(x + bias, approximate)
-  def FastGelu(x:Tensor, bias:Tensor|None=None):
-    # this is tanh approximated
-    return (x + bias).gelu() if bias is not None else x.gelu()
-  # TODO: fix this
-  def PRelu(X:Tensor, slope:Tensor):
-    slope = slope[0] if slope.shape[-1] != X.shape[-1] else slope
-    return (X > 0).where(X, X * slope)
+  def FastGelu(x:Tensor, bias:Tensor|None=None): return (x + bias).gelu() if bias is not None else x.gelu() # this is tanh approximated
```
Comment (chenyuxyz) [inline extra/onnx.py:293]: `((x + bias) if bias is not None else x).gelu()`, but w/e
